src/Resources/config/routing.yml

```diff
@@ -1,19 +1,3 @@
-sylius_refund_order_refunds_list:
-    path: /admin/orders/{orderNumber}/refunds
-    defaults:
-        _controller: Sylius\RefundPlugin\Action\OrderRefundsListAction
-
-sylius_refund_refund_units:
-    path: /admin/orders/{orderNumber}/refund-units
-    defaults:
-        _controller: Sylius\RefundPlugin\Action\RefundUnitsAction
-
-sylius_refund_order_credit_memos_list:
-    path: /admin/orders/{orderNumber}/credit-memos
-    defaults:
-        _controller: Sylius\RefundPlugin\Action\OrderCreditMemosListAction
-
-sylius_refund_credit_memo_details:
-    path: /admin/orders/{orderNumber}/credit-memos/{id}
-    defaults:
-        _controller: Sylius\RefundPlugin\Action\CreditMemoDetailsAction
+sylius_refund:
+    resource: "routing/admin.yml"
```

Comment:
Could we follow PluginSkeleton conventions? https://github.com/Sylius/PluginSkeleton/blob/master/src/Resources/config/admin_routing.yml